packages/mdc-select/_mixins.scss

```diff
@@ -89,22 +91,36 @@
   }
 }
 
-@mixin mdc-select-corner-radius($radius) {
-  border-radius: $radius $radius 0 0;
+@mixin mdc-select-shape-radius($radius, $rtl-reflexive: false) {
+  @if length($radius) > 2 {
+    @error "Invalid radius: '#{$radius}' component doesn't allow customizing all corners";
+  } @else if length($radius) == 2 {
+    $radius: nth($radius, 1) nth($radius, 2) 0 0;
+  } @else {
+    $radius: $radius $radius 0 0;
+  }
+
+  &, &__native-control {
+    @include mdc-shape-radius(mdc-shape-resolve-percentage-radius($mdc-shape-height, $radius), $rtl-reflexive);
+  }
 }
 
-@mixin mdc-select-outline-corner-radius($radius) {
+@mixin mdc-select-outline-shape-radius($radius, $rtl-reflexive: false) {
   // NOTE: idle and notched state border radius mixins
   // are broken into 2 different mixins, otherwise
   // we would be overly specific (big no, no). The cause of
   // this is because .mdc-notched-outline and .mdc-notched-outline__idle
   // are siblings. .mdc-notched-outline__idle needs to be a child of
   // .mdc-notched-outline in order to remedy this issue.
   .mdc-notched-outline {
-    @include mdc-notched-outline-corner-radius($radius);
+    @include mdc-notched-outline-shape-radius(mdc-shape-resolve-percentage-radius($mdc-shape-height, $radius), $rtl-reflexive);
   }
 
-  @include mdc-notched-outline-idle-corner-radius($radius);
+  @include mdc-notched-outline-idle-shape-radius(mdc-shape-resolve-percentage-radius($mdc-shape-height, $radius), $rtl-reflexive);
+
+  &__native-control {
```

Comment:
Why was this added?

Reply:
This'll ensure that it removes the native select element from click target. Previously this wan't happening in mixins but this is set by default.